index.html

```diff
@@ -4,44 +4,161 @@
         <p class="description">I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-            don’t hesitate to contact me.</p>
-            <nav class="icon-wrap">
-                <ul class="navicon">
-                   <li><a href="https://github.com/Zuwairanajma"><img class="github" src="PImages/GitHub.png" alt="Github Icon"></a></li>
-                   <li><a href="https://www.linkedin.com/in/zuwaira-sadiq-566b891b0?"><img class="linkedin" src ="PImages/Linkedin.png" alt = "Linkedin Icon"></a></li> 
-                   <li><a href="#"><img class ="angeli" src ="PImages/Vector3.png" alt="Angellist Icon"></a></li>
-                   <li><a href="https://twitter.com/@SadiqJuwairiyya"><img class ="twitter" src ="PImages/Twitter.png" alt ="Twitter Icon"></a></li>
-                   <li><a href="#"><img class ="medium" src="PImages/Vector5.png" alt ="Medium Icon"></a></li>
-                </ul>
-            </nav>
+            don’t hesitate to contact me.
+        </p>
+        <nav class="icon-wrap">
+            <ul class="navicon">
+                  <li><a href="https://github.com/Zuwairanajma"><img class= "github" src="PImages/GitHub.png" alt="Github Icon"></a></li>
+                  <li><a href="https://www.linkedin.com/in/zuwaira-sadiq-566b891b0?"><img class= "linkedin"src="PImages/Linkedin.png" alt="Linkedin Icon"></a></li> 
+                  <li><a href="#"><img class= "angeli"src="PImages/VectorH.png" alt="Angellist Icon"></a> </li>
+                  <li><a href="https://twitter.com/@SadiqJuwairiyya"><img class= "twitter" src="PImages/Twitter.png" alt="Twitter Icon"></a></li>
+                  <li><a href="#"><img class= "medium" src="PImages/Medium.png" alt="Medium Icon"></a></li>
+            </ul>
+        </nav>
         </div>
     </section>
     <!-- Headline Section End -->
-    </main>
+  
+    <!-- Recent Works Section Start -->
+
+    <section id="My-Recent" class="My-Recent">
+        <div id="recent-works_container" class="recent-works">
+        <h2 id="my-work" class="my-work">My Recent Works</h2>
+      </div>
+      <div class="line-divider">
+        <hr>
+  </div>
+      <div>
+      <img class = "placeholder" src="PImages/Img Placeholder.png" alt=""></div>
+          <div id="multip" class="multipost">
+              <h3 id="multi" class="multi">Multi-Post Stories</h3>
+              <p id="stories" class="stories">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p></div>
+          <ul id="buttons1" class="buttonss1">
+             <li class="css-stories">CSS</li>
+             <li class="html-stories">HTML</li>
+             <li class="boots-stories">BootStrap</li>
+             <li class="ruby-stories">Ruby</li>
+          </ul>
```

Comment:
- [ ] Nice work so far 👍🏿 on implementing the tech stack sections, but upon inspections I notice your design is overflowing to the left, please do well to adjust the width of the page.
 
![image](https://github.com/Zuwairanajma/Portfolio-Projects/assets/68823331/55066cd1-6424-40f2-b5cf-596676310fd2)


- [ ] Figma Design
![image](https://github.com/Zuwairanajma/Portfolio-Projects/assets/68823331/2d616c15-277d-40a5-9eca-3ec97f9b3ac7)